### PR TITLE
Add test for motion code in both pick and stow tasks

### PR DESCRIPTION
**Pull request by wkentaro**
We should test motion code to avoid bugs.
I fixed typo in euslisp code, and added tests for both stow/pick tasks.

### CODE REVIEW

**Comment by knorth55** [comment]
thx. sorry.

**Comment by knorth55** [comment]
I think `interactive_marker` and `publish_bin_boxes` should be in `baxter.launch` because it is environment settings.
Also, it is easier for test, too.

And the problem is tote boxes in stow task.
In stow task, we don't have `publish_tote_boxes` node, so it should be added to `baxter.launch`.

**Comment by knorth55** [comment]
currently, tote bounding boxes are published from `AttentionClipper` in `setup_for_stow.launch`.

**Comment by knorth55** [comment]
What I mean is in https://github.com/wkentaro/jsk_apc/pull/9.
please take a look.

**Comment by wkentaro** [comment]
> The same as above?

Sorry?

File: jsk_arc2017_baxter/test/test-stow-arc-interface.l [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+(require "package://jsk_arc2017_baxter/euslisp/lib/arc-interface.l")
+
+(init-unit-test)
+
+(jsk_arc2017_baxter::arc-init)
+
+(deftest test-stow-arc-interface
+  ;; :recognize-tote-boxes
+  (send *ti* :recognize-tote-boxes)
+  (send *ti* :recognize-tote-boxes)
```
Comment (knorth55) [inline jsk_arc2017_baxter/test/test-stow-arc-interface.l]: @pazeshun you mean here right? sorry it's my fault.

Comment (pazeshun) [inline jsk_arc2017_baxter/test/test-stow-arc-interface.l]: Oh, I see.
Sorry for my poor comment...

Comment (knorth55) [inline jsk_arc2017_baxter/test/test-stow-arc-interface.l]: I made a PR https://github.com/wkentaro/jsk_apc/pull/10.
thx!

**Comment by wkentaro** [comment]
@knorth55 Could you please review this?

**Comment by 708yamaguchi** [comment]
Please add how to use

**Comment by pazeshun** [comment]
I'll add

**Pull request merged**